docs/introduction.rst

```diff
@@ -2,50 +2,34 @@
 Introduction
 ============
 
-The dimod API provides a binary quadratic :term:`model` (BQM) class that contains
-:term:`Ising` and quadratic unconstrained binary optimization (\ :term:`QUBO`\ ) models
-used by :term:`sampler`\ s such as the D-Wave system.
-It provides utilities for constructing new samplers and :term:`composed sampler`\ s.
-It also provides useful functionality for working with these models and samplers.
-Its reference examples include several samplers and composed samplers.
+The dimod API provides a binary quadratic model (BQM) class that contains
```

Comment:
I think here it shouldn't be "the dimod API", maybe just "dimod"

Reply:
Updated to dimod.